test/test_lomap.py

```diff
@@ -122,27 +121,24 @@ def test_get_set_add(self):
         with self.assertRaises(ValueError):
             inst.__add__('no_mol_obj')
 
-
     # Check serial and parallel mode
     def test_serial_parallel(self):
         logging.basicConfig(format='%(message)s', level=logging.CRITICAL)
-        db = DBMolecules('test/basic')
+        db = DBMolecules(_rf('basic/'))
         s_strict, s_loose = db.build_matrices()
-        db.options.paralell = multiprocessing.cpu_count()
+        db.options['parallel'] = multiprocessing.cpu_count()
         p_strict, p_loose = db.build_matrices()
         
-        assert (all(s_strict == p_strict))
-        assert (all(s_loose == p_loose))
-
+        assert (s_strict == p_strict).all()
+        assert (s_loose == p_loose).all()
 
     def test_read_mol2_files(self):
         logging.basicConfig(format='%(message)s', level=logging.CRITICAL)
-        db = DBMolecules('test/basic')
-        db.options.directory = 'test/'
+        db = DBMolecules(_rf('basic/'))
+        db.options['directory'] = _rf('')
         with self.assertRaises(IOError):
             db.read_molecule_files()
 
-
     # Test which heterocycles I can grow (growing off a phenyl)
     # Test by Max indicates that growing complex heterocycles tends
     # to fail, so only allow growing phenyl, furan and pyrrole
```

Comment:
These comments can be removed, it's a Cresset artefact and likely very SOMD (or single topology-)specific. There should be some possibility for users to add customisation like this for themselves.